knexfile.js

```diff
@@ -0,0 +1,46 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
```

Comment:
This is a really clean way to have all of your knex stages broken up by being commented out. I'm trying to think of a way you could leave these all un-commented and just do some sort of a check in your `dbConfig.js` file. Like the line `module.exports = knex(knexConfig.development);` do something like `module.exports = 'this is the condition I'm having trouble with' ? knex(knexConfig.development): knex(knexConfig.staging);`